source/loop/statsd.rst

```diff
@@ -0,0 +1,34 @@
+======
+Statsd
+======
+
+Loop-Server have got a number of Statsd counters and timers that can
```

Comment:
I would suggest creating a table, with "name", "type" & "description" instead of several sections